src/Sarif.UnitTests/Transformers/SarifVersionOneToCurrentTests.cs

```diff
@@ -32,6 +57,47 @@ private static void VerifyVersionOneToCurrentTransformation(string v1LogText, st
             v2LogText.Should().Be(v2LogExpectedText);
         }
 
+        private static void VerifyVersionOneToCurrentTransformationFromResourceFile(string v1InputResourceFileName, string v2ExpectedResourceFileName)
```

Comment:
>ResourceFile [](start = 71, length = 12)

Nit: I suggest that wherever you use `ResourceFile` in a name, you instead use `EmbeddedResource` (since the code actually deals in embedded resources rather than loose files).